authorization-service/dotenv.config.js

```diff
@@ -0,0 +1,13 @@
+const dotenv = require('dotenv');
```

Comment:
it's possible to pass env variables to the application even without the 'dotenv' module. Serverless supports env files out of the box - https://www.serverless.com/framework/docs/environment-variables/

Reply:
ok, thanks